scripts/tasks/src/api-extractor.ts

```diff
@@ -6,7 +6,7 @@ import chalk from 'chalk';
 import { isCI } from 'ci-info';
 import * as glob from 'glob';
 import { ApiExtractorOptions, TaskFunction, apiExtractorVerifyTask, logger, series, task } from 'just-scripts';
-import type * as ApiExtractorTypes from 'just-scripts/src/tasks/apiExtractorTypes';
+import type * as ApiExtractorTypes from '@microsoft/api-extractor';
```

Comment:
previous import path not accessible anymore, we can import directly from api-extractor

Reply:
hope those are the same types, just-scripts came with various invalid types ( for typescript configs for example etc )